### PR TITLE
Password reset

### PR DESCRIPTION
**Pull request by amefire**
UI of  the Password Reset process.

### CODE REVIEW

File: ResetPassword.tsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+  submitButton: {
+    backgroundColor: "#0000FF",
+    padding: 10,
+    margin: 15,
+    height: 40,
+  },
+  submitButtonText: {
+    color: "white",
+  },
+  appTitleLarge: {
```
Comment (omefire) [inline ResetPassword.tsx:150]: It seems like there are styles you aren't using here. Please, clean it up if that's the case.

File: SendConfirmationEmail.tsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+      >
+        Please enter your email to receive a password reset link:
+      </Text>
+    </View>
+    <Formik
+      initialValues={{
+        email: "",
+      }}
+      onSubmit={(values, actions) => {
+        alert(JSON.stringify(values));
```
Comment (omefire) [inline SendConfirmationEmail.tsx:47]: Axe this.

File: SignUp.tsx [hunk 3]
```diff
@@ -0,0 +1,244 @@
+import React from "react";
+import {
```
Comment (omefire) [inline SignUp.tsx:2]: The Signup page shouldn't be part of this PR.

File: App.tsx [hunk 4]
```diff
@@ -8,7 +8,7 @@
  * @format
  */
 
-import React from 'react';
```
[the remark this answers is not in the document]
Comment (amefire) [reply App.tsx:11]: I do use '',  but whenever i save, it changes to " "

File: SendConfirmationEmail.tsx (hunk 2 above)
Comment (amefire) [reply SendConfirmationEmail.tsx:47]: what do you mean ?
